source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -83,13 +86,53 @@ the SRV record. For example, when using the non-FQDN ``servername`` and the
 default domain set to ``example.com``, a Client should end up requesting the
 SRV record for ``servername.example.com``.
 
+Default Connection String Options
+---------------------------------
+
+A Client MUST also query the DNS server for TXT records. If available, these
+TXT records provide default connection string options. In most cases only one
+TXT record is necessary, unless the default options exceed the maximum length
+of 255 characters that TXT records allow for. To allow for this case, a Client
+MUST allow for multiple TXT records for the same host name and consider the
+options in all of them.
+
+Information returned with each TXT record is a simple URI string, just like
+the options in a connection string.
+
+Although options in a connection string are separated by a ``&``, this
+character MUST NOT be present to separate options between TXT record strings
+as a separation is implied by using multiple TXT records.
+
+TXT records MAY be queried before SRV records because for example they could
+include the ``replicaSet`` option to define the replica set name.
```

Comment:
I agree that TXT records MAY be queried either before or after SRV records. But the fact that "replicaSet" could be included isn't relevant. Clients MUST query both the SRV and the TXT records before attempting any connection to MongoDB; therefore it doesn't matter which order they're queried in. Doesn't make a difference if there's a "replicaSet" option in particular.

Reply:
I realise it wasn't relevant. I had just included it here as an example. Will remove it.